py4DSTEM/process/calibration/ellipse.py

```diff
@@ -63,7 +63,7 @@ def fit_ellipse_1D(ar, center=None, fitradii=None, mask=None):
     rr = np.sqrt((xx - x0) ** 2 + (yy - y0) ** 2)
     _mask = (rr > ri) * (rr <= ro)
     if mask is not None:
-        _mask *= mask is False
+        _mask *= mask == False  # noqa: E712
```

Comment:
This should be converted to a binary logical operation. I think this is effectively doing `logical_and`?

Reply:
I was just changing this back to the state before #558, and added a comment to not raise an error in linters.